###  [:house:返回首頁](https://github.com/ourhimalayas/txt)
---

## 【重磅】數十位科學家聯名致信著名醫學雜誌質疑病毒起源（中英文對照全文）
`2020-09-06 08:25 Isaiah4031` [轉載自GNews](https://gnews.org/zh-hant/337117/)

![](https://s3.amazonaws.com/gnews-media-offload/wp-content/uploads/2020/09/06080427/SARS-CoV-2_LTE-photo.jpg)圖片來源：[Fusion Medical Animation](https://unsplash.com/@fusion_medical_animation?utm_source=unsplash&amp;utm_medium=referral&amp;utm_content=creditCopyText) on [Unsplash](https://unsplash.com/s/photos/virus?utm_source=unsplash&amp;utm_medium=referral&amp;utm_content=creditCopyText)
新聞簡述：9月3日著名醫學期刊《醫學病毒學》（Journal of Medical Virology）發表了數十位來自世界各地的科學家聯名致信該期刊主編的經同行評議過的信件。該文對新型冠狀病毒來源於蝙蝠冠狀病毒，特別是RaTG13的行業共識提出了質疑。新型冠狀病毒是第七種人冠狀病毒，但也是第一個且是唯一一個具備大流行潛力的人冠狀病毒。然而，其宿主取向/適應模式與其它冠狀病毒存在顯著差異，這就引起了關於新型冠狀病毒的近端起源的討論。

[下載英文原文](https://onlinelibrary.wiley.com/doi/epdf/10.1002/jmv.26478) （譯者注：所有內容均以英文原文為準）

[下載中英文對照版（PDF）](https://s3.amazonaws.com/gnews-media-offload/wp-content/uploads/2020/09/06093731/SARS-CoV-2_LTE-traditional.pdf)

**Letter to the Editor**

**致編者信函**

**Questions concerning the proximal origin of SARS-CoV-2**

**關於新型冠狀病毒近端起源的問題**

This article has been accepted for publication and undergone full peer review but has not been through the copyediting, typesetting, pagination and proofreading process, which may lead to differences between this version and the Version of Record. Please cite this article as doi:10.1002/jmv.26478.

這篇文章已被接收出版，並通過完整的同行評審，但未經編輯、排版、分頁和校對等過程，這可能會導致該版本與原版間產生差異。請將該文章引用為doi:10.1002/jmv.26478。

Keywords: Coronavirus, Genetic Variability, Mutation, Fusion Protein

關鍵詞 ：冠狀病毒，遺傳變異，突變，融合蛋白

**Summary:**

**總結：**

There is a consensus that Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) originated naturally from Bat coronaviruses (CoVs), in particular RaTG13. However, the SARS-CoV-2 host tropism/adaptation pattern has significant discrepancies compared to other CoVs, raising questions concerning the proximal origin of SARS-CoV-2. The flat and non-sunken surface of the sialic acid-binding domain of SARS-CoV-2 spike protein (S protein) conflicts with the general adaptation and survival pattern observed for all other CoVs. Unlike RaTG13, SARS-CoV-2 recombination presumably occurred between the S1/S2 domains of S protein enabling host furin protease utilization. Although millions of recorded cases have been recorded globally, SARS-CoV-2 S protein does not have any apparent further recombination, placing it in conflict with the recombination models of other CoVs. Similarly, the S protein receptor-binding domain (RBD) of SARS-CoV-2 has not accumulated high-frequency non-synonymous substitutions, differentiating SARS-CoV-2 from other CoVs that have positive selection/adaptation mutations in their RBDs.

有一個共識：嚴重急性呼吸綜合徵冠狀病毒2 (SARS-CoV-2)自然起源於蝙蝠冠狀病毒(CoVs)，特別是RaTG13。然而，與其它冠狀病毒相比，SARS-CoV-2的宿主取向/適應模式存在顯著差異，這就引發了關於SARS-CoV-2近端起源的問題。SARS-CoV-2刺突蛋白(S蛋白)的唾液酸結合結構域平坦而非凹陷的表面與所有其它冠狀病毒的一般適應性和生存模式相違背。與RaTG13不同的是，SARS-CoV-2重組可能發生在S蛋白的S1/S2結構域之間，從而可以利用宿主弗林蛋白酶。儘管全球記錄了數百萬病例，但SARS-CoV-2 S蛋白沒有出現任何明顯的進一步重組，這使其與其它冠狀病毒的重組模型相違背。同樣，SARS-CoV-2的S蛋白受體結合域(RBD)並未積累高頻率非同義替代，這將SARS-CoV-2與其他RBDs選擇/適應性突變的冠狀病毒區別開來。

**Discussion**

**討論**

Andersen and colleagues documented the possible natural origin of SARS-CoV-2 from BatCoV RaTG13 1,2. SARS-CoV-2 is the seventh zoonotic CoV virus capable of infecting humans, but the first and only human coronavirus (HCoV) with pandemic potential 3. Bat or rodent CoVs demonstrate certain specific changes in the S protein receptor-binding domain (RBD), as well as the S protein glycan-binding N-terminal domain (NTD), during host tropism/adaptation 4,5. SARS-CoV-2, unlike other CoVs, does not have those signature changes, suggesting that these RBD and NTD subdomains are of very recent origin.

Andersen和他的同事記錄了SARS-CoV-2可能的自然來源是蝙蝠冠狀病毒 RaTG13 1,2。SARS-CoV-2是第七種能夠感染人類的人畜共患冠狀病毒，但也是第一個且是唯一一個具備大流行潛力的人冠狀病毒(HCoV) 3。蝙蝠或嚙齒類動物的冠狀病毒在宿主向性/適應過程中，S蛋白受體結合域(RBD)和S蛋白多糖結合N端結合域(NTD)發生了特定的變化4,5。與其它冠狀病毒不同，SARS-CoV-2沒有這些特徵變化，這表明這些RBD和NTD子域是最近才出現的。

The “Canyon Hypothesis” explains the development of canyons, depression zones or cavities on the surfaces of influenza virus, human rhinovirus, and Meningo viruses 6 . In CoVs (except SARS-CoV-2), the S protein NTD domain has several predicted glycanbinding domains, with a common feature being the hidden localization of these glycan binding domains to cavities to limit their access to antibodies and immune cells 5 . This pattern of CoVs is thought to be an evolutionary measure to restrict the recognition of these active sites by host immune system 4 .

「峽谷假說」解釋了流感病毒、人類鼻病毒和腦膜炎病毒表面的峽谷、凹陷區或空腔的發展 6。在冠狀病毒(SARS-CoV-2除外)中,S蛋白NTD結構域有幾個可預測多糖結合域,其共同特徵是這些多糖結合域在腔中的隱蔽定位從而限制它們與抗體和免疫細胞接觸 5。這冠狀病毒模式被認為是一種進化措施，以限制宿主免疫系統識別這些活性位點 4。

HCoVs can evade detection by host glycan-binding immune receptors. Comparative genomic analysis of six HCoVs with their corresponding native bat or rodent CoVs suggests compatibility with the “Canyon Hypothesis” resulting from various adaptive S protein NTD non-synonymous mutations near or at the glycan binding domain which are predicted to result in these NTD domains being hidden below the protein surface5 . The predicted flat, non-sunken pattern of the SARS-CoV-2 S protein NTD glycan binding domains conflicts with this evolutionary host tropism/adaptation strategy 7.

人冠狀病毒可以逃避宿主聚糖結合免疫受體的檢測。對六種人冠狀病毒及其相應的當地蝙蝠或嚙齒動物冠狀病毒進行比較基因組分析表明，與多糖結合域附近的各種適應性S蛋白NTD非同義突變導致的「峽谷假說」具有相融性，預計這些突變會導致這些NTD域被隱藏在蛋白質表面之下5。預測的SARS-CoV-2 S蛋白NTD聚糖結合域的預測平坦、非凹陷模式與這種進化宿主趨向性/適應策略相違背7。

A template-switching mechanism is presumably responsible for the high rate of RNA recombination in CoVs. In host cells, CoV RNAs show discontinuous RNA synthesis materialized by pauses of the RNA-dependent complex and subsequent jumps to downstream template acceptor sequences. This process results in subgenomic minus strand RNAs which serve as templates for subgenomic mRNAs. Due to the mechanistic similarity to recombination, this process might be at the origin of recombinant CoVs co-opting other CoV or even host related sequences8. Instances include the mouse hepatitis coronavirus S protein NTD sialic acid binding domain, likely arising from recombination of viral RNA with human galectin RNA sequences 8.

一種模板轉換機制可能是冠狀病毒（CoVs）與 RNA高重組率的原因。在宿主細胞中，冠狀病毒RNA呈現出不連續的RNA合成，表現為依賴RNA的復合物間歇並隨後跳轉到下游模板受體序列。這一過程產生亞基因組負鏈RNA，作為亞基因組mRNA的模板。由於重組機制的相似性，這一過程可能是起源於重組冠狀病毒（CoV）吸收其它冠狀病毒（CoV）甚至宿主相關的序列8。例如，小鼠肝炎冠狀病毒S蛋白NTD唾液酸結合域，可能由病毒RNA與人乳糖蛋白RNA序列重組而產生8。

The furin recognition motif present at the SARS-CoV2 S1/S2 junction has no analogy in other “linage B” beta-coronaviruses, including neither pangolin-CoV nor RaTG13 1 . This indicates that the S protein S1/S2 junction is not a hot spot for RNA recombination termination that depends on a pattern swapping templates (copy-choice) 8 . Additionally, clinical isolates of SARS-CoV-2 S protein have not indicated any further recombination in this S1/S2 area, suggesting that the addition of a motif for S1/S2 site furin cleavage constituted a unique recombination occurrence. Finally, the CoV-unique insertion of 4 amino-acids creating a novel RRAR furin cleavage site introduces two arginine codons CGG-CGG, whose usage is extremely rare in CoVs, further supporting the hypothesis of a unique recombination occurrence.

在SARS-CoV2 S1/S2接合處存在的弗林蛋白酶識別基序在其他「linage B」-冠狀病毒中沒有類比，包括穿山甲冠狀病毒和RaTG13 1。這說明S蛋白S1/S2連接並不是RNA重組終止的熱點，RNA重組終止依賴於模式交換模板(copy-choice) 8。此外，臨床分離到的SARS-CoV-2 S蛋白在這個S1/S2區域沒有任何進一步的重組，這表明為S1/S2位點添加的基序弗林蛋白酶裂解構成了一個獨特的重組。最後，4個氨基酸在冠狀病毒（CoV）中獨特的插入，形成了一個新的RRAR弗林蛋白酶裂解位點，引入了兩個精氨酸密碼子CGG-CGG，這兩個精氨酸密碼子在冠狀病毒（CoVs）中使用極為罕見，進一步支持了發生獨特重組的假設。

HCoVs have high-frequency “hot spots” for non-synonymous amino acid replacements that can possibly create positive selection for host tropism/adaptation, resistance to neutralizing antibodies, or immune evasion 2 . Interestingly, clinical SARS-CoV-2 isolates to date have only a single high frequency non-synonymous mutation, D614G, in their S protein 9. Based on currently known mutation rates and patterns in clinical isolates of SARS-CoV-2, the S protein does not appear to be a mutational “hot spot” for SARSCoV-2, unlike other human CoVs.

人冠狀病毒（HCoVs）為非同義氨基酸提供高頻替換「熱點」，這些替換可能會趨向宿主趨同性/適應、而對中和抗體或免疫逃避產生抵抗2。有趣的是，迄今為止臨床分離的SARS-CoV-2在其S蛋白中只有一個高頻非同義突變，D614G9。對臨床分離株SARS-CoV-2，根據目前已知的變異率和模式, 不像其它人冠狀病毒，S蛋白似乎沒有SARS CoV-2突變的「熱點」。

SARS-CoV-2 is the seventh HCoV, but the first HCoV with pandemic potential. SARSCoV disappeared without a pandemic, and MERS-CoV is mostly endemic to the Arabian Peninsula with some additional limited traveler infections resulting in outbreaks in South Korea 3,4. These unique features of SARS-CoV-2 raise several questions concerning the proximal origin of the virus that require further discussion.

SARS-CoV-2是第七種人冠狀病毒，卻是第一個具有大流行潛力的人冠狀病毒。非典在大流行前就消失了，MERS-CoV主要是阿拉伯半島的地方傳染病，另外還有部分有限的旅行者感染導致韓國爆發3,4。SARS-CoV-2的這些獨特特徵引發幾個有關該病毒近端起源的問題，還需進一步討論。

**Author Contributions:**

**作者貢獻**

M. S. conceived the study.

A. M. B, V. N. U, A. L., and K. L. provided critical review.

N. R., D. P., A. L. S. P. S.. V. N. U, K. L.T. M. A. edited the article,

M. S., P. A., T. M. A. formatted the article.

All authors M. S., D. P., P. A., T. M.A., S. S. H., A. S., R. K., K. L., D. M., M. T., A. L., P. P. C., V. N. U., S. P. S., B. D. U., N. R., A. M. B. and G. Z. K. of the consortium interpreted the results and approved the final version for submission.

M. S. 構思該研究.

A. M. B, V. N. U, A. L., and K. L. 進行重要審稿

N. R., D. P., A. L. S. P. S.. V. N. U, K. L.T. M. A.撰寫文章,

M. S., P. A., T. M. A. 排版該文章

所有作者（M. S., D. P., P. A., T. M.A., S. S. H., A. S., R. K., K. L., D. M., M. T., A. L., P. P. C., V. N. U., S. P. S., B. D. U., N. R., A. M. B. and G. Z. K. ）聯合解釋該結果並同意提交最終版本。

**Competing Interests statement:**The authors declare no competing interests.

**利益衝突聲明：**作者聲明沒有利益衝突。

**References****：**

**參考文獻：**

1. Andersen KG, Rambaut A, Lipkin WI, Holmes EC, Garry RF. The proximal origin of SARS-CoV-2.NatMed.2020;26(4):450-452.doi:10.1038/s41591-020-0820-9;

2. Malaiyan J, Arumugam S, Mohan K, Gomathi Radhakrishnan G. An update on the

origin of SARS-CoV-2: Despite closest identity, bat (RaTG13) and pangolin derived

coronaviruses varied in the critical binding site and O-linked glycan residues [published online ahead of print, 2020 Jul 7]. J Med Virol. 2020;10.1002/jmv.26261. doi:10.1002/jmv.26261

3. Forni D, Cagliani R, Clerici M, Sironi M. Molecular Evolution of Human Coronavirus Genomes.Trends Microbiol. 2017;25(1):35-48.

doi:10.1016/j.tim.2016.09.001

4. Hulswit RJ, de Haan CA, Bosch BJ. Coronavirus Spike Protein and Tropism Changes. Adv Virus Res. 2016;96:29-57. doi:10.1016/bs.aivir.2016.08.004

5. Li F. Receptor recognition mechanisms of coronaviruses: a decade of structural

studies. J Virol. 2015;89(4):1954-1964. doi:10.1128/JVI.02615-14

6. Rossmann MG. The canyon hypothesis. Hiding the host cell receptor attachment

site on a viral surface from immune surveillance. J Biol Chem.1989;264(25):14587- 14590.

7. Fantini J, Di Scala C, Chahinian H, Yahi N. Structural and molecular modelling

studies reveal a new mechanism of action of chloroquine and hydroxychloroquine

against SARS-CoV-2 infection. Int J Antimicrob Agents. 2020;55(5):105960.

doi:10.1016/j.ijantimicag.2020.105960

8. Makino S, Keck JG, Stohlman SA, Lai MM. High-frequency RNA recombination of murine coronaviruses. J Virol. 1986;57(3):729-737. doi:10.1128/JVI.57.3.729-737.1986

9. Brufsky A. Distinct viral clades of SARS-CoV-2: Implications for modeling of viral spread [published online ahead of print, 2020 Apr 20]. J Med Virol.

2020;10.1002/jmv.25902. doi:10.1002/jmv.25902

翻譯：【重生】【一花一世界】 校對：【重生】【一花一世界】編輯：【Michelle】

戰友之家玫瑰園小隊出品

0
